apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9707,6 +9707,13 @@ input.mat-input-element{
     .delete-modal-overlay {
         z-index: 9992; /*for delete custom zIndex modal*/
     }
+    #ledger-transaction-left-list sh-select::ng-deep input,
```

Comment:
`::ng-deep` is deprecated, please refer the doc for reference: https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

> The shadow-piercing descendant combinator is deprecated and support is being removed from major browsers and tools. As such we plan to drop support in Angular (for all 3 of /deep/, >>> and ::ng-deep). Until then ::ng-deep should be preferred for a broader compatibility with the tools.